### ISSUE TITLE
option to unload all packages

### ISSUE THREAD

**Issue by colinpmillar** [ices-tools-prod/icesTAF#14]
This would be to clean out all packages before running a script.  There are a bunch of occurances of functions being masked by other packages, such as raster::select and dpyr::select 

Something like this could be useful:

```r
detachAllPackages <- function() {
  basic.packages <- 
  c("package:stats", "package:graphics", "package:grDevices", 
     "package:utils", "package:datasets", "package:methods", "package:base")
  package.list <- search()[ifelse(unlist(gregexpr("package:", search())) == 1, TRUE, FALSE)]
  package.list <- setdiff(package.list, basic.packages)
  if (length(package.list) > 0) {
    for (package in package.list) 
      detach(package, character.only = TRUE)
  }
}
```

**Comment by colinpmillar** [comment]
We might think about adding an optiong to `sourceTAF(taf = TRUE)` and `sourceAll(taf = TRUE)` a bit like `cran = TRUE` in the check packages functions in devtools.  If `taf = TRUE` it would mimic what is done on the server and clears out all the folders, unloads packages etc.

**Comment by arni-magnusson** [comment]
Implemented in 8a8731e.
